ogs6py/ogs.py

```diff
@@ -445,34 +446,67 @@ def run_model(self, **args):
             Name of the file to write STDOUT of ogs
             Default: out
         path : `str`, optional
-            Path of the directory in which the ogs executable can be found
+            Path of the directory in which the ogs executable can be found.
+            If ``container=True``: Path to the directory in which the 
+            Singularity executable can be found
+        container: `boolean`, optional
+            Switch to run OGS in a singularity container. 
+            If True, `container_path` is required
+            Default: False
+        container_path : `str`, optional
+            Path of the OGS container file. Only required if ``container=True``
         args : `str`, optional
             additional arguments for the ogs executable
         """
         ogs_path = ""
+        container_path = ""
         if self.threads is None:
             env_export = ""
         else:
             env_export = f"export OMP_NUM_THREADS={self.threads} && "
+        if "container" in args:
```

Comment:
I think the boolean container variable is redundant. I think it is enough to check for container_path.